protos/telemetry/telemetry.proto

```diff
@@ -452,6 +509,29 @@ message GpsInfo {
     FixType fix_type = 2; // Fix type
 }
 
+/*
+ * Raw GPS information type.
+ *
+ * Warning: this is an advanced type! If you want the location of the drone, use
+ * the position instead. This message exposes the raw values of the GNSS sensor.
+ */
+ message RawGps {
+     uint64 timestamp_us = 1; // Timestamp in microseconds (UNIX Epoch time or time since system boot, to be inferred)
+     double latitude_deg = 2; // Latitude in degrees (WGS84, EGM96 ellipsoid)
+     double longitude_deg = 3; // Longitude in degrees (WGS84, EGM96 ellipsoid)
+     float absolute_altitude_m = 4; // Altitude AMSL (above mean sea level) in metres
+     uint32 eph = 5; // GPS HDOP horizontal dilution of position (unitless). If unknown, set to: UINT16_MAX
```

Comment:
Hang on. This should be called what it actually is. I think the comment is correct and not the variable name, hence it is `hdop` and I'm pretty sure it should be a float, and divide the number coming in via mavlink by 100, looking at https://github.com/PX4/PX4-Autopilot/blob/c2151cb4fafc8fdbbf136d6da4989185b0763217/src/modules/mavlink/mavlink_messages.cpp#L1393.

Unknown can then be `NaN`.